src/bereikbaarheid/resources/utils.py

```diff
@@ -73,6 +73,21 @@ def create_dataset(self, in_stream, **kwargs):
         kwargs["format"] = "csv"
         return tablib.import_set(in_stream, **kwargs)
 
+    def can_export(self):
+        return super().can_export()
+
+
+    def export_data(self, dataset, **kwargs):
+        # remove the deprecated `escape_output` param if present
```

Comment:
waarom zou deze param er zijn als hij deprecated is? wat context?

Reply:
added comment